apps/server/src/modules/provisioning/strategy/sanis/sanis.strategy.ts

```diff
@@ -109,12 +112,23 @@ export class SanisProvisioningStrategy extends SchulconnexProvisioningStrategy {
 					}
 				);
 
-				const schulconnexLizenzInfoResponses = plainToClass(
+				const schulconnexPoliciesInfoResponse = plainToClass(
 					SchulconnexPoliciesInfoResponse,
 					schulconnexPoliciesInfoAxiosResponse
 				);
-				await this.checkResponseValidation(schulconnexLizenzInfoResponses);
-				externalLicenses = SchulconnexResponseMapper.mapToExternalLicenses(schulconnexLizenzInfoResponses);
+
+				await this.checkResponseValidation(schulconnexPoliciesInfoResponse);
+
+				const schulconnexPoliciesInfoLicenceResponses: SchulconnexPoliciesInfoLicenseResponse[] =
+					schulconnexPoliciesInfoResponse.data.filter((item): item is SchulconnexPoliciesInfoLicenseResponse => {
+						if (item instanceof SchulconnexPoliciesInfoErrorResponse) {
```

Comment:
I think chaning the logic to `if item instanceof SchulconnexPoliciesInfoLicenseResponse return true` is more safe for the future